indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/SinglePhaseSubTask.java

```diff
@@ -651,7 +654,8 @@ private Map<String, TaskReport> getTaskCompletionReports()
                 getTaskCompletionRowStats(),
                 errorMsg,
                 false, // not applicable for parallel subtask
-                segmentAvailabilityWaitTimeMs
+                segmentAvailabilityWaitTimeMs,
+                ingestedIntervals
```

Comment:
The subtasks don't need to condense intervals, because the parent task gathers all intervals from subtasks, then condenses them.  See `ParallelIndexSupervisorTask` line 1249.

Reply:
Got it! Thanks.